sbin/jtreg.sh

```diff
@@ -91,7 +92,11 @@ settingUpEnvironmentVariablesForJTREG()
 runJtregViaMakeCommand()
 {
   echo "Running jtreg via make command (debug logs enabled)"
-  make test jobs=10 LOG=debug  
+  if [ -z $JTREG_TEST_SUBSETS }; then
```

Comment:
Currently, when we run the jtreg without options, we default to jdk_core and langtools (the latter doesn't seem to work, but jdk_core has enough coverage that we don't mind atm).

We could default to jdk_core, which gives us a core set of tests and removes the non-functional langtools, or we could switch to jdk_all. I don't know how long jdk_all takes, but I'm guessing it's longer than jdk_core.

The latter already takes 25 minutes on our ultra-fast linux server.